src/creates/set_value.js

```diff
@@ -0,0 +1,63 @@
+const { APIFY_API_ENDPOINTS } = require('../consts');
+const { wrapRequestWithRetries } = require('../request_helpers');
+const { getOrCreateKeyValueStore } = require('../apify_helpers');
+
+const setValue = async (z, bundle) => {
+    const { storeIdOrName, key, value } = bundle.inputData;
+    const store = await getOrCreateKeyValueStore(z, storeIdOrName);
+    const keyValueStoreValueUrl = `${APIFY_API_ENDPOINTS.keyValueStores}/${store.id}/records/${key}`;
+
+    let valueObject;
+    try {
+        valueObject = JSON.parse(value);
```

Comment:
You can throw even before if value is not a string.

Reply:
It is always a string, it was validated on Zapier UI as I set up to the field to be `type: 'text'`